python-phoenixdb/.gitignore

```diff
@@ -8,6 +8,5 @@
 .tox
 dev-support/artifacts
 dev-support/work
-phoenixdb/.eggs
-phoenixdb/build
-phoenixdb/e
+.eggs
```

Comment:
keep the old entries instead of replacing, some build scenarios may generate files there